doc/src/manual/performance-tips.md

```diff
@@ -1075,16 +1075,16 @@ with
 
 ```jldoctest prealloc
 julia> function xinc!(ret::AbstractVector{T}, x::T) where T
-           ret[1] = x
```

Comment:
When there wasn't a loop it felt ok to "hardcode" the length to 3 but now it seems better to use the length of `ret`?

Reply:
agreed